.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &linux_defaults
   working_directory: /go/src/github.com/cosmos/cosmos-sdk
   docker:
-    - image: circleci/golang:1.11.5
+    - image: circleci/golang:1.12.1
```

Comment:
This is a sign that the branch hasn't been rebased on top of `cosmos/cosmos-sdk`'s latest `develop`.

Reply:
rebased with latest changes

Reply:
Why is this still showing up in the diff then?

Reply:
I suspect that your fork's `develop` branch is still behind cosmos-sdk's `develop`. Please sync the two, then rebase

Reply:
my bad.
It was already up to date.
I needed to force push the changes (now the files changes and commits are correct)